lua/iswap/internal.lua

```diff
@@ -166,6 +170,24 @@ function M.swap_nodes_and_return_new_ranges(a, b, bufnr, should_move_cursor)
   return { a_data, b_data }
 end
 
+function M.move_nodes_to_index(children, cur_node, cur_node_idx, bufnr, a_idx, config)
```

Comment:
why is `nodes` plural in the name of this function? Isn't it just one node?

Reply:
Oh yeah you're right, not sure what I was thinking, maybe because I saw `swap_nodes_and...` above